### PR TITLE
Feature/ Add merge page to confirm profile merge

### PR DESCRIPTION
**Pull request by xkopenreview**
this pr should
1. add a merge page for user to confirm profile merge
2. remove prefill logic for profile merge modal

related to https://github.com/openreview/openreview-api/pull/392

### CODE REVIEW

**Comment by melisabok** [comment]
I think the UI should the same one than is shown in the alternate confirm. If the UI doesn't know the profile that are going to be merged then I would just show the same message the we show in this PR: https://github.com/openreview/openreview-web/pull/1859

Screen Shot 2024-04-16 at 2 19 03 PM

**Comment by melisabok** [comment]
If a login with any of both profiles, the profile that is confirming the email and the profile that contains the  confirmed email, it let me click on "Submit". 

Shouldn't we let them click on "Submit" only with the profile that is confirming the email? cc @carlosmondra

**Comment by melisabok** [comment]
Is it possible to show the email next to the profile id?

Screen Shot 2024-04-18 at 4 15 36 PM

**Review by melisabok — APPROVED** [review]

File: pages/profile/merge.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  if (error) return <ErrorDisplay statusCode={error.statusCode} message={error.message} />
+  if (!user || !profileToMergeFrom) return <LoadingSpinner />
+
+  return (
+    <>
+      <header>
+        <h1>Profile Merge</h1>
+      </header>
+      <p className="mt-4">
+        Click Confirm Profile Merge button below to confirm merging{' '}
```
Comment (zbialecki) [inline pages/profile/merge.js:74]:
```suggestion
        Click the confirm button below to merge{' '}
```

File: pages/profile/merge.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        <a
+          href={`/profile?id=${profileToMergeFrom.profileId}`}
+          target="_blank"
+          rel="noreferrer"
+        >
+          <strong>{profileToMergeFrom.profileId}</strong>
+        </a>
+        <strong>{`<${profileToMergeFrom.email}> `}</strong>
+        to your profile.
```
Comment (zbialecki) [inline pages/profile/merge.js:83]:
```suggestion
        into your user profile.
```

File: pages/profile/merge.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+      </p>
+      <p className="mb-4">
+        Please contact{' '}
+        <a href="mailto:info@openreview.net" target="_blank" rel="noreferrer">
+          info@openreview.net
+        </a>{' '}
+        if you didn&apos;t request for this profile merge.
```
Comment (zbialecki) [inline pages/profile/merge.js:90]:
```suggestion
        if you didn&apos;t request for this profile to be merged with your account.
```

**Review by zbialecki — COMMENTED** [review]
Some text suggestions.

**Review by zbialecki — APPROVED** [review]

**Comment by zbialecki** [comment]
Looks like there's a test failing here... an error message might have been changed.

**Pull request merged**